### ISSUE TITLE
Change icons?

### ISSUE THREAD

**Issue by joq3** [nfarina/homebridge#996]
Hi,

I use Homebridge together with Harmony Hub and Broadlink RM 3 plugins, and all the switches are shown in Apple Home app as a normal Switch icon. Is it possible to configure this and specify which icon that should be used? Like lightbulb icons and such, that way it is easier to differente them from eachother.

Another thing, is it possible to add a switch which is stateless? I have some IR-codes that are the same code for on and off, and if I use my remote to turn off the TV, the Home app will still show it as on.

Thank you!

**Comment by AlexGustafsson** [comment]
In the "Home" app you can change some settings such as the Icon. Be sure to check there to see if it is what you need.

**Comment by vswraith** [comment]
Is it possible to put custom icons in the Home app?

**Comment by AlexGustafsson** [comment]
@vswraith, yes, you can change icons in the app. 

1. In the app, hold down on an item.
2. Tap "Details" (the only button visible, furthest down the screen). 
3. Tap the icon next to the device's name and change the icon.
4. _Press "Done" in the top right corner._
5. _Press "Done" again._
6. _Tap anywhere to exit details mode._

Steps written in italic are optional.

**Comment by SkyJohn** [comment]
That only works for lights doesn't it? And then you only get 4 different icons.

I don't get other icon options for things like power points, fans, sensors, etc...

**Comment by vswraith** [comment]
Yeah, I can change some basic icons,but is it possible to add custom icons?

**Comment by KhaosT** [comment]
No

**Comment by SkyJohn** [comment]
Could you add icons on a jailbroken device, or is it something that has to be done on Apple's end?
